src/site/content/en/learn-web-vitals/lab-and-field-data-differences/index.md

```diff
@@ -0,0 +1,386 @@
+viewport, but that largest element can change as the page is loaded or if new
+content gets dynamically added to the viewport.
+
+In the lab, the browser will wait until the page is fully loaded before
+determining what the LCP element was. But in the field, the browser will [stop
+monitoring](/lcp/#when-is-largest-contentful-paint-reported) for larger elements
+after the user scrolls or interacts with the page.
+
+This makes sense (and is necessary) because users typically will wait to
+interact with a page until it "appears" loaded, which is exactly what the LCP
+metric aims to detect. It also wouldn't make sense to consider elements added to
+the viewport after a user interacts because those elements might have only been
+loaded _because_ of something the user did.
+
+The implication of this, though, is that field data for a page may report faster
+LCP times, depending on how users behave on that page.
+
+### FID
+
+#### FID requires real-user interaction
+
+The [FID](/fid/) metric measures how responsive a page is to user interactions,
+_at the time when users actually chose to interact with it._
+
+The second part of that sentence is critical because lab tests, even those that
+support script user behavior, cannot accurately predict when users will choose
+to interact with a page, and thus cannot accurately measure FID.
+
+#### TBT and TTI do not consider user behavior
+
+Lab metrics like [Total Blocking Time (TBT)](/tbt/) and [Timg to Interactive
+(TTI)](/tti/) are intended to help diagnose issues with FID because they
+quantify how much the main thread is blocked during page load.
+
+The idea is that pages with lots of synchronous JavaScript or other intensive
+rendering tasks are more likely to have a blocked main thread when the user
+first interacts. However, if users wait to interact with the page until after
+the JavaScript finishes executing, FID may be very low.
+
+When users will choose to interact with a page depends largely on whether or not
+it _looks_ interactive, and this cannot be measured with TBT or TTI.
+
+#### TBT and TTI do not consider tap delay
+
+If a site is not optimized for mobile viewing, browsers will [add a 300ms
+delay](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away)
+after any tap before running event handlers. They do this because they need to
+determine whether the user is trying to double-tap to zoom before they can fire
+mouse or click events.
+
+This delay counts toward a page's FID because it contributes to the real input
+latency that users experience. But since this delay is not technically a [Long
+Task](https://w3c.github.io/longtasks/), it doesn't affect a page's TBT or TTI.
+This means a page may have poor FID despite having very good TBT and TTI scores.
+
+{% Aside %}
+**Tip:** pages can avoid the tab delay issue by always specifying a [mobile viewport](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away).
```

Comment:
```suggestion
To avoid the tab delay issue on a page, always specify a [mobile viewport](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away).
```